docs/import/manage_disaster.html

```diff
@@ -110,6 +110,8 @@
         </table>
       </div>
       <div><label for="damage-asset-select">Damage asset: </label><select id="damage-asset-select"></select>
+        <div id="map-bounds-div" hidden>
+         OR (before damage asset available)<br/><label for="map-bounds-sw">Southwest damage bounds, in "lat, lng" format: </label><input id="map-bounds-sw"><br/><label for="map-bounds-sw">Northeast damage bounds, in "lat, lng" format: </label><input id="map-bounds-ne"></div>
```

Comment:
I think it would be helpful to have a literal example the way we do (probably unnecessarily) for the add disaster fields.

Reply:
This look ok?